package.json

```diff
@@ -21,9 +27,10 @@
     "webpack": "^1.13.3"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "eslint src && react-scripts start",
```

Comment:
We don't want to do this because this means run eslint, and if it succeeds, run react-scripts start.  I think we need a separate lint watch command.